docs/contributing/governance/veyfi.md

```diff
@@ -0,0 +1,56 @@
+# veYFI
+
+> This product is not deployed yet and is still under construction! You can follow development at https://github.com/yearn/veYFI
```

Comment:
@saltyfacu do you think merging this PR would be an issue even with this disclaimer that the product is not live yet? Just pinging to be sure this PR stays on-hold :)

Reply:
yeah, I'd like it to be finished. We can coord with the team building it. I'd also like someone else to review this for accuracy